src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexCultureTests.cs

```diff
@@ -247,12 +238,12 @@ public static IEnumerable<object[]> Match_In_Different_Cultures_TestData()
                     // Expected answers in the default en-US culture
                     yield return new object[] { "(?i:I)", option, engine, enUS, "xy\u0131ab", "" };
                     yield return new object[] { "(?i:iI+)", option, engine, enUS, "abcIIIxyz", "III" };
-                    yield return new object[] { "(?i:iI+)", option, engine, enUS, "abcIi\u0130xyz", "Ii\u0130" };
-                    yield return new object[] { "(?i:iI+)", option, engine, enUS, "abcI\u0130ixyz", "I\u0130i" };
+                    yield return new object[] { "(?i:iI+)", option, engine, enUS, "abcIi\u0130xyz", "Ii" };
+                    yield return new object[] { "(?i:iI+)", option, engine, enUS, "abcI\u0130ixyz", "\u0130i" };
                     yield return new object[] { "(?i:iI+)", option, engine, enUS, "abc\u0130IIxyz", "\u0130II" };
                     yield return new object[] { "(?i:iI+)", option, engine, enUS, "abc\u0130\u0131Ixyz", "" };
                     yield return new object[] { "(?i:iI+)", option, engine, enUS, "abc\u0130Iixyz", "\u0130Ii" };
-                    yield return new object[] { "(?i:[^IJKLM]I)", option, engine, enUS, "ii\u0130i\u0131ab", "" };
+                    yield return new object[] { "(?i:[^IJKLM]I)", option, engine, enUS, "ii\u0130i\u0131ab", "\u0130i" };
```

Comment:
happened to notice that `private const char Kelvin_sign = '\u212A';` further down is unused.